src/status_im/multiaccounts/update/core.cljs

```diff
@@ -58,7 +33,8 @@
                  [{:method "settings_saveSetting"
                    :params [setting setting-value]
                    :on-success on-success}]}
-                (when (#{:name :photo-path :prefered-name} setting)
+                (when (and (not dont-sync?)
```

Comment:
not dont-sync? :D

Reply:
yes, is a bit silly, but I need a negative, as 99% of the time we want to sync, only when reacting to a syncing event we don't want to sync :) anything better we could name it?